make/tasks/Makefile.muxed

```diff
@@ -0,0 +1,18 @@
+# Copyright 2004-2015 Cray Inc.
+# Other additional copyright holders may be indicated within.
+#
+# The entirety of this work is licensed under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+CHPL_MAKE_THREADS=soft-threads
```

Comment:
Is this what you had in mind for how to define CHPL_MAKE_THREADS?

Also, is it ok to have Makefile.muxed here, or should it live in the internal repo?